https/setMatches.js

```diff
@@ -1,5 +1,5 @@
 const functions = require("firebase-functions");
-const { setAllRandomParticipant } = require("../utils/utils");
+const { matchAllParticipants } = require("../utils/matchAllParticipants");
```

Comment:
Out of curiousity why did you split out teh utility file? I thought its purporse was to keep the functions together?

Reply:
The util file has grown so big it is no longer easy to read. It's a better practice to split it up. I prefer to have one utility file for each function or area of functionality. E.g. one for randomness, one for string functions, etc.